chicago/views.py

```diff
@@ -1,32 +1,34 @@
 from django.shortcuts import render
 from django.conf import settings
 from django.http import Http404, HttpResponsePermanentRedirect
-from django.core.urlresolvers import reverse
+from django.urls import reverse
 
 from datetime import date, timedelta, datetime
 
 from chicago.models import ChicagoBill, ChicagoEvent
-from councilmatic_core.models import Action
 from councilmatic_core.views import *
 
 from haystack.query import SearchQuerySet
 
+from django.db.models import DateTimeField
+from django.db.models.functions import Cast
+
 class ChicagoIndexView(IndexView):
     template_name = 'chicago/index.html'
     bill_model = ChicagoBill
     event_model = ChicagoEvent
 
     def last_meeting(self):
-        return ChicagoEvent.objects.filter(name__exact=settings.CITY_COUNCIL_MEETING_NAME)\
-                           .filter(start_time__lt=datetime.now())\
-                           .latest('start_time')
+        return ChicagoEvent.objects.annotate(start_date_dt=Cast('start_date', DateTimeField())).filter(name__exact=settings.CITY_COUNCIL_MEETING_NAME)\
```

Comment:
it would be better to move this annotation into django councilmatic, if not possible to move into Chicago model.

Reply:
done! one already existed upstream, so i used that instead.